lib/components/button_unit.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:maui/components/responsive_grid_view.dart';
+
+class Buttonunit extends StatefulWidget {
```

Comment:
Do not create a new Button.
Make use of UnitButton which is already there for this purpose.
When we are displaying numbers, display the dots also